reference_data/management/tests/update_hpo_tests.py

```diff
@@ -1,132 +1,132 @@
-    u'property_value: http://purl.org/dc/terms/license https://hpo.jax.org/app/license xsd:string\n',
-    u'logical-definition-view-relation: has_part\n',
-    u'\n',
-    u'[Term]\n',
-    u'id: HP:0000001\n',
-    u'name: All\n',
-    u'comment: Root of all terms in the Human Phenotype Ontology.\n',
-    u'xref: UMLS:C0444868\n',
-    u'\n',
-    u'[Term]\n',
-    u'id: HP:0000002\n',
-    u'name: Abnormality of body height\n',
-    u'def: "Deviation from the norm of height with respect to that which is expected according to age and gender norms." [HPO:probinson]\n',
-    u'synonym: "Abnormality of body height" EXACT layperson []\n',
-    u'xref: UMLS:C4025901\n',
-    u'is_a: HP:0000003 ! Growth abnormality\n',
-    u'created_by: peter\n',
-    u'creation_date: 2008-02-27T02:20:00Z\n',
-    u'\n',
-    u'[Term]\n',
-    u'id: HP:0000003\n',
-    u'name: Multicystic kidney dysplasia\n',
-    u'alt_id: HP:0004715\n',
-    u'def: "Multicystic dysplasia of the kidney is characterized by multiple cysts of varying size in the kidney and the absence of a normal pelvicaliceal system. The condition is associated with ureteral or ureteropelvic atresia, and the affected kidney is nonfunctional." [HPO:curators]\n',
-    u'comment: Multicystic kidney dysplasia is the result of abnormal fetal renal development in which the affected kidney is replaced by multiple cysts and has little or no residual function. The vast majority of multicystic kidneys are unilateral. Multicystic kidney can be diagnosed on prenatal ultrasound.\n',
-    u'synonym: "Multicystic dysplastic kidney" EXACT []\n',
-    u'synonym: "Multicystic kidneys" EXACT []\n',
-    u'synonym: "Multicystic renal dysplasia" EXACT []\n',
-    u'xref: MSH:D021782\n',
-    u'xref: SNOMEDCT_US:204962002\n',
-    u'xref: SNOMEDCT_US:82525005\n',
-    u'xref: UMLS:C3714581\n',
-    u'is_a: HP:0000001 ! Renal cyst\n',
-    u'\n',
-    u'[Term]\n', # no is_a
-    u'id: HP:0000005\n',
-    u'name: Mode of inheritance\n',
-    u'alt_id: HP:0001453\n',
-    u'alt_id: HP:0001461\n',
-    u'def: "The pattern in which a particular genetic trait or disorder is passed from one generation to the next." [HPO:probinson]\n',
-    u'synonym: "Inheritance" EXACT []\n',
-    u'xref: UMLS:C1708511\n',
-    u'\n',
-    u'[Term]\n', # is_a == "HP:0000118"
-    u'id: HP:0000152\n',
-    u'name: Abnormality of head or neck\n',
-    u'def: "An abnormality of head and neck." [HPO:probinson]\n',
-    u'synonym: "Abnormality of head or neck" EXACT layperson []\n',
-    u'synonym: "Head and neck abnormality" EXACT layperson []\n',
-    u'xref: UMLS:C4021817\n',
-    u'is_a: HP:0000118 ! Phenotypic abnormality\n',
+    'format-version: 1.2\n',
+    'data-version: hp/releases/2020-03-27\n',
+    'saved-by: Peter Robinson uni\xe7\xf8de\n, Sebastian Koehler, Sandra Doelken, Chris Mungall, Melissa Haendel, Nicole Vasilevsky, Monarch Initiative, et al.\n',
```

Comment:
does the real data ever have unicode?

Reply:
Up to now, I have found that `dbNSFP4.0_gene` and `hp.obo` downloaded from the network have unicode chars in them. I'm not sure about `omim` data because it need a key to download. For the rest of the reference data, there are no unicode chars in them.

Reply:
In `hp.obo`, there is a line:
```
property_value: http://purl.org/dc/elements/1.1/creator "Sebastian Köhler" xsd:string
```